lib/periodicChecks.js

```diff
@@ -174,7 +175,38 @@ const ensureAllIssuesHaveProjects = async (context) => {
   });
 };
 
+/**
+ * 1. Check if the new issue have a project assigned.
+ * 2. If not then it will ping Core Maintainers
+ * @param {import('probot').Context} context
+ */
+const ensureNewIssuesHaveProjects = async(context) =>{
+  await sleep(20000);
+  const issue = context.payload.issue;
+  const allProjectCards = await getAllProjectCards(context);
+
+  const doesIssueHaveProject = allProjectCards.some((card) => {
+    return card.content_url.endsWith(issue.number);
+  });
+
+  // doesIssueHaveProject is a boolean Variable which will be true if issue
+  // Have Project Assigned, Else False
```

Comment:
This comment is not needed, the purpose of `doesIssueHaveProject` is pretty clear.

Reply:
Ok Sir, Removing it in latest commit.

Reply:
Sure Sir, Adding them right now.